src/Eshopworld.Tests.Core/Eshopworld.Tests.Core.csproj

```diff
@@ -26,6 +26,7 @@
   </ItemGroup>
   
   <ItemGroup>
+    <PackageReference Include="Eshopworld.Core" Version="4.2.1" />
```

Comment:
is it ok to add this reference so I can add extension method for BB?

Reply:
I see no issues. Eshopworld.Core does not reference this one so we're not going into a circular dep ref